### PR TITLE
SPSH-1004 Refactor PersonenkontextEvents

### PR DESCRIPTION
**Pull request by marode-cap**
# Description
- POST /dbiam-personenkontexte uses the PersonenkontexteUpdate aggregate to add new kontext
- Removes the create- and deletePersonenkontextEvents
- Removes the utility module (Great idea and should be kept in mind, but with the changes to the events it's not needed right now)

- PersonDeletedEvent is not removed, because sometimes we actually care if a person is actually deleted and didn't just lose their kontexte

## Approval for review
- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

### CODE REVIEW

File: src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts [hunk 1]
```diff
@@ -257,13 +253,7 @@ export class DBiamPersonenkontextRepo {
         );
 
         await this.em.persistAndFlush(personenKontextEntity);
-        this.eventService.publish(
```
Comment (pkleybolte) [inline src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts:260]: Es sollte das PersonenkontextUpdatedEvent geworfen werden. Kein Event zu werfen ist keine Option.

File: src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts [hunk 2]
```diff
@@ -302,13 +292,7 @@ export class DBiamPersonenkontextRepo {
             mapAggregateToData(personenKontext),
         );
         await this.em.persistAndFlush(personenKontextEntity);
-        this.eventService.publish(
```
Comment (pkleybolte) [inline src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts:305]: Es sollte das PersonenkontextUpdatedEvent geworfen werden. Kein Event zu werfen ist keine Option.

File: src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts [hunk 3]
```diff
@@ -346,14 +330,6 @@ export class DBiamPersonenkontextRepo {
             organisationId: organisationId,
             rolleId: rolleId,
         });
-        this.eventService.publish(
```
Comment (pkleybolte) [inline src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts:349]: Es sollte das PersonenkontextUpdatedEvent geworfen werden. Kein Event zu werfen ist keine Option.

**Review by pkleybolte — CHANGES_REQUESTED** [review]

File: src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts (hunk 1 above)
Comment (marode-cap) [reply src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts:260]: Deprecated

File: src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts (hunk 3 above)
Comment (marode-cap) [reply src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts:349]: Deprecated

File: src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts (hunk 2 above)
Comment (marode-cap) [reply src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts:305]: Deprecated

**Review by pkleybolte — APPROVED** [review]

**Pull request merged**